recipes/galera.rb

```diff
@@ -118,14 +118,15 @@
 
   execute 'correct-debian-grants' do
     # Add sensitive true when foodcritic #233 fixed
-    command 'mysql -r -B -N -e "GRANT SELECT, INSERT, UPDATE, DELETE, '\
-      'CREATE, DROP, RELOAD, SHUTDOWN, PROCESS, FILE, REFERENCES, INDEX, '\
-      'ALTER, SHOW DATABASES, SUPER, CREATE TEMPORARY TABLES, '\
-      'LOCK TABLES, EXECUTE, REPLICATION SLAVE, REPLICATION CLIENT, '\
-      'CREATE VIEW, SHOW VIEW, CREATE ROUTINE, ALTER ROUTINE, '\
-      "CREATE USER, EVENT, TRIGGER ON *.* TO '" + \
-      node['mariadb']['debian']['user'] + \
-      "'@'" + node['mariadb']['debian']['host'] + "' IDENTIFIED BY '" + \
+    command "mysql -r -B -N -u root "\
+      "--password='" + node['mariadb']['server_root_password'] + "' -e " \
```

Comment:
I'm sorry for that, but if the password is empty, it fail on some conditions. you need to find a way to use a condition here to add the --password option only if password is not empty